BepInEx.Core/Utility.cs

```diff
@@ -221,6 +221,54 @@ public static bool TryResolveDllAssembly<T>(AssemblyName assemblyName,
         return false;
     }
 
+    /// <summary>
+    ///     Try to resolve the symbols data from an assembly DLL.
+    /// </summary>
+    /// <param name="dllPath">File path of the DLL assembly.</param>
+    /// <param name="symbolsData">The symbols of the DLL assembly.</param>
+    /// <returns>True, if the symbols were found. Otherwise, false.</returns>
+    public static bool TryResolveAssemblySymbols(string dllPath, out byte[] symbolsData)
+    {
+        symbolsData = null;
+        
+        var subDirectory = Path.GetDirectoryName(dllPath);
+
+        var potentialSymbolsFileNames = new[]
+        {
+            $"{dllPath}.mdb",
```

Comment:
Sidenote-ish:
Let's say a plugin wants to be universal and work (with debug symbols) on legacy mono, modern mono and coreclr platforms that have mdb, mdb/pdb and pdb support respectively.
Always preferring mdb over pdb will lead to: no symbols on coreclr, potentially worse symbols (if ppdb had more information than mdb, although I don't think that is the case today) on modern mono and only legacy mono would work as expected.

Reversing the order today would break legacy mono altogether so... I think it's fine as is, and we could reverse the order after adding the automatic debug symbols conversion that would figure out "the right thing" for the current runtime.